File: discover_ruby/write_them_to_disc.rb
#!/usr/bin/ruby
require 'HTTPClient'

extheaders = {
  'User-Agent' => 'Holberton_school',
  'Authorization' => '5b0fbcd1114ea6d6c375c414f1d48d87ef0b71bb'
}

http = HTTPClient.new

api = http.get_content ('https://api.github.com/search/repositories?q=language:ruby&sort=stars&order=desc')

File.write('/tmp/27', api)


puts "The File was saved!"

File: discover_more_python/write_them_to_disc.py
from urllib2 import Request, urlopen, URLError

request_headers = {
  'User-Agent': 'Holberton_School',
  'Authorization': 'token 7e573dd60371dc91c35929139cdca698e477e85c'
}

request = Request('https://api.github.com/search/repositories?q=language:python&sort=stars&order=desc')

try:
	response = urlopen(request)
	read_file = response.read()
        f = open('/tmp/27','w')
        f.write(read_file);
        f.close()
        print "The file was saved!"
except URLError, e:
    print 'No file. Got an error code:', e
